modules/sdk/gradle-plugins-source-formatter/README.markdown

```diff
@@ -44,13 +40,11 @@ Name | Depends On | Type | Description
 ---- | ---------- | ---- | -----------
 `formatSource` | - | [`FormatSourceTask`](#formatsourcetask) | Runs Liferay Source Formatter to format the project files.
 
-## Task Types
-
-### <a name="formatsourcetask"></a>`FormatSourceTask`
```

Comment:
I added the anchor so the table row at https://github.com/jhinkey/liferay-portal/pull/366/commits/25339df998c6eda21675da68518d9403396c05df#diff-ccc8745ed2498ddec5f3e4edca062ad2R41 can have a hyperlink. If you don't want it, we should remove the link as well!